### ISSUE TITLE
`GET /parties/AUTH_SERVICE/{participantId}` design

### ISSUE THREAD

**Issue by lewisdaly** [mojaloop/pisp#83]
__We have decided to use the ALS to store the dfspId.__

This deviates from the [existing usage of `GET /parties`](https://docs.mojaloop.io/mojaloop-specification/documents/API%20Definition%20v1.0.html#63-api-resource-parties), so we need a new design.

The main things to consider are:
1. How does the response body 
2. How do we maintain/record _reserved_ namespaces in the ALS? 
    - In this example, we propose using the TYPE of `AUTH_SERVICE`, but there may be instances where we adopt this pattern in other parts of Mojaloop

**Comment by elnyry-sam-k** [comment]
Hi @lewisdaly - what are the data items / fields we're looking for in a response to this GET call?

**Comment by lewisdaly** [comment]
I've since done a little more research, and I think we want `GET /participants` and not get parties.

See [this example](https://github.com/mojaloop/pisp/blob/master/docs/linking/README.md#161-requesting-a-challenge) from the linking flows. 

Internally, the 3p-api-adapter does a `GET /participants/CONSENTS/{id}` to lookup the participantId of the auth service responsible for the given consent. The response back is just a `PUT /participants/CONSENTS/{id}`, with a body of: `{"fspId": "central-auth"}`

**Comment by elnyry-sam-k** [comment]
Thanks Lewis, yes agree `GET /participants` seems more suitable for this.

**Comment by lewisdaly** [comment]
Yeah I think we tend to conflate the ALS with `GET /parties` where in actual fact it's more storing participants.

In terms of designs, this leaves us wide open to implement something like `GET /consents/{id}` in the future, and we can adopt the same pattern as we have today for `GET /parties` (first lookup the participant, then ask that participant for the information)

So I guess we will need a new Oracle for the `consentId -> participantId` mapping as well.

**Comment by Akshanshkaushal** [comment]
@lewisdaly     I would love to work upon the issue , could you please assign this ?
